.travis.yml

```diff
@@ -2,12 +2,14 @@ language: ruby
 cache: bundler
 rvm:
   - 2.6.2
+addons:
+  chrome: stable
 services:
   - postgresql
 before_script:
   - psql -c 'create database indyhackers_test;' -U postgres
   - cp config/database.yml.travis config/database.yml
 
 script:
-  - RAILS_ENV=test bundle exec rake --trace db:migrate spec
-  - RAILS_ENV=test bundle exec rake --trace features
+  - bundle exec rake --trace db:setup
+  - bundle exec rake --trace spec
```

Comment:
I could probably have this be one command, but I figured it would allow us to have other "steps" we can run.